rewrite-gradle/src/main/java/org/openrewrite/gradle/AddGradleWrapper.java

```diff
@@ -73,10 +74,13 @@ public String getDescription() {
     Validated gradleWrapper;
 
     @Override
-    public Validated validate() {
+    public Validated validate(ExecutionContext ctx) {
         if (gradleWrapper == null) {
-            gradleWrapper = super.validate().and(GradleWrapper.validate(version, distribution,
-                    new HttpUrlConnectionSender(Duration.ofSeconds(3), Duration.ofSeconds(10))));
+            HttpSender httpSender = ctx.getMessage("httpSender");
```

Comment:
Can we namespace this property `org.openrewrite.httpSender`?

Reply:
Thanks. View added.